client/src/components/AnnouncementCreationModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+import { useNavigate } from "react-router-dom";
+import { postData } from "../api";
+import { useAuth } from "../AuthContext";
+import "../Announcement.css"
+
+export default function ClubCreationModal(props) {
+  const [announcementTitle, setTitle] = useState("");
+  const [announcementText, setDesc] = useState("");
+  const { user } = useAuth();
+
+
+  const onSubmit = async () => {
+    const userId = user?.id;
+    const clubId = props.clubId
+    let announcementTime = new Date()
```

Comment:
We can actually just have the [database automatically do timestamping for us](https://github.com/varangill/ClubHub/blob/master/server/src/database/migrations/20240220194148_clubs-timestamp.ts) which we should prob do here to be consistent with the other tables